tests/test_core.py

```diff
@@ -317,6 +317,18 @@ def test_filter_day(test_df, test_day):
         assert unique_time[0] == expected
 
 
+def test_filter_numpy_vals(test_df):
```

Comment:
Can this have a more verbose/descriptive name?

Reply:
ok